.tekton/central-db-pull-request.yaml

```diff
@@ -0,0 +1,434 @@
+apiVersion: tekton.dev/v1
```

Comment:
The newer `apiVersion`, `on-cel-expression` instead of the trigger was preset by RHTAP's generated PR.